citi-onboarding/capibaribe-racing
# Feature/backers section

## What I did
implemented section to store images of sponsors


- Thing I’ve done
- 3 fields have been added per section topic
      The first is the name ID of the topic,
     The second is a brand description field.
     The third an image of the brand

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout <branch name>`
   1. `git pull origin <branch name>`
   1. `yarn install`
   1. `yarn start`

1. In your browser go to <http://localhost>:port

1. Check if the component looks just like the mockup in different screen sizes

1. Read the changed files section on GitHub

1. Test the things that you think are worth testing, even the ones that are not in this description
